numpy/core/tests/test_einsum.py

```diff
@@ -491,16 +491,24 @@ def check_einsum_sums(self, dtype, do_opt=False):
         assert_array_equal(np.einsum('ij,ij->j', p, q, optimize=True),
                            [10.] * 2)
 
-        p = np.ones((1, 5))
-        q = np.ones((5, 5))
+        # a blas-compatible contraction broadcasting case which was failing
+        # for optimize=True (ticket #10930)
+        x = np.array([0., 0.])
```

Comment:
Can we use array of say 5s just incase we hit some sort of weird bug.

Reply:
Sure, I'll have to wait until I'm home from work.